src/System.ServiceModel.Syndication/tests/System.ServiceModel.Syndication.Tests.csproj

```diff
@@ -17,11 +16,17 @@
     <Content Include="$(MsBuildThisFileDirectory)\TestFeeds\AtomFeeds\*.xml">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+	<Content Include="$(MsBuildThisFileDirectory)\TestFeeds\RssFeeds\*.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Please change all `always` to `preservenewest`.

Reply:
Done.